healthzed/endpoint.py

```diff
@@ -13,15 +21,14 @@ def check_health():
 
 
 @app.post("/send_ping")
-def send_ping(data: PingRequest):
-    response = deliver_ping(
-        from_user=data.from_user,
-        to_user=data.to_user,
-        message=data.message,
-        phone_number=data.phone_number,
+async def send_ping(data: PingRequest):
+    send_task = asyncio.create_task(
+        deliver_ping(message=data.message, phone_number=data.phone_number)
     )
-    if response:
-        return PingResponse(status_code=200, message="Ping sent successfully!")
+
+    await send_task
```

Comment:
In a future PR, there will likely be a message service that will handle this kind of stuff, rather than having it live in the `endpoint.py` service directly.